libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -144,7 +144,7 @@ const uint32_t AP_GPS::_baudrates[] PROGMEM = {4800U, 38400U, 115200U, 57600U, 9
 
 // initialisation blobs to send to the GPS to try to get it into the
 // right mode
-const prog_char AP_GPS::_initialisation_blob[] PROGMEM = UBLOX_SET_BINARY MTK_SET_BINARY SIRF_SET_BINARY;
+const prog_char AP_GPS::_initialisation_blob[] PROGMEM = SBF_SETUP_MSG UBLOX_SET_BINARY MTK_SET_BINARY SIRF_SET_BINARY ;
 const prog_char AP_GPS::_initialisation_raw_blob[] PROGMEM = UBLOX_SET_BINARY_RAW_BAUD MTK_SET_BINARY SIRF_SET_BINARY;
```

Comment:
You need to configure the sbf here as well, unless there is a reason that ublox raw logging and the septentrio GPS are incompatible.